platform/core.multitabs/src/org/netbeans/core/multitabs/impl/TabDataRenderer.java

```diff
@@ -248,8 +255,10 @@ public void paint( Graphics g ) {
             }
 
             // paint tab decorators
-            for( TabDecorator td : decorators ) {
-                td.paintAfter( tabData, g, rect, isSelected );
+            if (!isHover) {
```

Comment:
Why are tab decorators not invoked in the isHover case? TabDecorator is a public API, so imagine having to describe this behavior in its Javadoc: "if the UIManager property nb.multitabs.hoverBackground is defined, then paintAfter is only called when the tab is not hovered over". That would be a really weird caveat.

Reply:
> Why are tab decorators not invoked in the isHover case?

The reason was that I wanted to avoid that the "project color bar" (from PR #2048) is painted for hovered tabs. Probably a bad idea. I'll change it back...